packages/shared/locales/en.json

```diff
@@ -1055,15 +1055,6 @@
                 "assetId": "Asset Id"
             }
         },
-        "importErc20Token": {
```

Comment:
Wouldn't it be necessary to remove it from all `locales`?

Reply:
we have automation with crowdin, when you touch a locale file you only need to edit the english file and then crowdin takes care of the rest, if we update all locales, crowdin can get confused